### DISCUSSION TITLE
能否考虑将home assistant内的其它平台设备同步至米家APP并进行控制

### DISCUSSION THREAD

**Discussion by HellyW** [XiaoMi/ha_xiaomi_home#54] [Ideas]
RT： 能否考虑将home assistant内的其它平台设备同步至米家APP并进行控制

**Comment by erchuyi** [comment]
疯了吗！(╯°Д°)╯

**Comment by LOVE2CMOL** [comment]
这个你就有点异想天开了 这工作量 无异于把整个core+所有要兼容的插件 搬过来

**Comment by HellyW** [comment]
类似于  巴法  插件， 将HA内设备同步到米家。  但巴法目前同步过来的只能让小爱同学控制，如果能够作为虚拟设备可以接入中枢网关，就可以利用米家自动化/米家极客版实现全屋智能的联动，将生态全部转移回米家, 而不是去适配home assistant

**Comment by HellyW** [comment]
应该说类似于Apple - home bridge 插件一样， 将 HA 的设备接入HomeKit中；

**Comment by VergilGao** [comment]
哥们你搞反了，HA才是那个中枢平台，米家只是HA其中的一部分。
这个集成最终的目的也是为了让HA能够纯本地控制所有的米家设备并且和其他品牌的设备联动。

**Comment by Hugh696** [comment]
这是讨论区，不是许愿池（滑稽
